dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -1121,4 +1125,62 @@ public ResponseEntityContainerView copy(@Context final HttpServletRequest reques
             throw new DoesNotExistException("Container with Id: " + id + " does not exist");
         }
     }
+
+    /**
+     * Deletes Container(s).
+     *
+     * This method receives a list of identifiers and deletes the containers.
+     * To delete a container successfully the user needs to have Edit Permissions over it.
+     * @param request            {@link HttpServletRequest}
+     * @param response           {@link HttpServletResponse}
+     * @param containersToDelete {@link String} container identifier to look for and then delete it
+     * @return Response
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @DELETE
+    @Path("bulkdelete")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
```

Comment:
I would prefer to make this async, but it is ok by now
Do not expect tons of objects deleted at the same time

Reply:
This could be easily handled by applying limits on front-end at the moment if it is really needed to limit